hms-api/src/handler/comment/create.ts

```diff
@@ -0,0 +1,19 @@
+import {buildResponse} from '../../rest/responses';
+import {createComment} from '../../service/idea_comment-service';
+import {wrapHandler} from '../handler-wrapper';
+import CommentCreateRequest from '../../rest/Comment/CommentCreateRequest';
+import CommentCreateResponse from '../../rest/Comment/CommentCreateResponse';
+
+export async function create(event, context, callback) {
+  await wrapHandler(async () => {
+    const request = CommentCreateRequest.parse(event.body);
+    const comment = await createComment(
+      request.ideaId,
+      request.userId,
+      request.text,
+      request.replyTo,
```

Comment:
replyTo sounds too much (exactly) like the email concept of `REPLY-TO` maybe something more like `parentCommentId` would be good (this would also allow a sort of infinite nesting of comments)